index.js

```diff
@@ -38,10 +38,17 @@ function parse(args, aliases, defaults, unknown, out) {
           }
         }
       } else {
-        var end = arg.slice(2).search(SHORTSPLIT) + 2
-        var value = end === arg.length
-          ? (j = i + 1) === len || "-" === args[j][0] || args[(i = j)]
-          : arg.slice(end)
+        SHORTSPLIT.lastIndex = 2
```

Comment:
@jridgewell Could we make this a separate PR please? 🙏 

Thanks.

Reply:
Do you want me to split all three up?

Reply:
Yes, please, that will help me get stuff reviewed and merged faster. Thanks.